tests/x11/mac80211_hwsim.pm

```diff
@@ -0,0 +1,219 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: mac80211 hw test
```

Comment:
Please add some more here, I know you can do :-) Describe the purpose of the test, limitations, etc.